catalog/src/main/kotlin/com/adevinta/spark/catalog/CatalogApp.kt

```diff
@@ -321,3 +331,5 @@ private val lightScrim = android.graphics.Color.argb(0xe6, 0xFF, 0xFF, 0xFF)
  * https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:activity/activity/src/main/java/androidx/activity/EdgeToEdge.kt;l=40-44;drc=27e7d52e8604a080133e8b842db10c89b4482598
  */
 private val darkScrim = android.graphics.Color.argb(0x80, 0x1b, 0x1b, 0x1b)
+
+internal val AppBasePath = "spark://catalog"
```

Comment:
I'm wondering if this `catalog` part is actually useful for something?

Reply:
Done in bc1dcd18a45a737e2f2a4245eb54f30529e1a4e1